concourse/pipelines/linux-image-build.jsonnet

```diff
@@ -385,16 +386,40 @@ local DebianImgPublishJob(image, env, workflow_dir) = imgpublishjob {
   image_prefix: common.debian_image_prefixes[image],
 };
 
-local ImgGroup(name, images) = {
-  name: name,
+local SapWorkloadTestJob(image) = sap_test.saptestjob {
+  image: image,
+};
```

Comment:
there's no logic in this function, so just replace it with direct use of the template